### PR TITLE
Document `kubernetes.io/bootstrapping` label / annotation

### PR DESCRIPTION
**Pull request by NitishKumar06**
Fix: #35602

### CODE REVIEW

File: content/en/docs/reference/labels-annotations-taints/_index.md [hunk 1]
```diff
@@ -114,6 +114,17 @@ to the name of the namespace. You can't change this label's value.
 This is useful if you want to target a specific namespace with a label
 {{< glossary_tooltip text="selector" term_id="selector" >}}.
 
+### kubernetes.io/bootstrapping
+
+Example: ` kubernetes.io/bootstrapping=rbac-defaults`
+
+Used on: Clusters
```
Comment (reylejano) [inline content/en/docs/reference/labels-annotations-taints/_index.md:121]: I don't think `Used on: Clusters` is correct
```suggestion
Used on: Clusters
```

**Comment by tengqm** [comment]
/hold

We have to understand the label/annotation well before documenting it. What is the purpose of this label? What are the value options?

**Comment by sftim** [comment]
/retitle Document `kubernetes.io/bootstrapping` label / annotation
/sig auth

File: content/en/docs/reference/labels-annotations-taints/_index.md (hunk 1 above)
Comment (NitishKumar06) [reply content/en/docs/reference/labels-annotations-taints/_index.md:121]: @reylejano the following page (https://docs.openshift.com/container-platform/4.7/authentication/using-rbac.html) demonstrates that this annotation is used to view the cluster roles etc.

Comment (reylejano) [reply content/en/docs/reference/labels-annotations-taints/_index.md:121]: ClusterRoles and Clusters are not the same

File: content/en/docs/reference/labels-annotations-taints/_index.md [hunk 2]
```diff
@@ -114,6 +114,17 @@ to the name of the namespace. You can't change this label's value.
...
+
+This annotation is used to view the cluster roles and bindings.
+
+To view the cluster roles and their associated rule sets:
+
+``$ oc describe clusterrole.rbac``
```
Comment (reylejano) [inline content/en/docs/reference/labels-annotations-taints/_index.md:127]: we should use kubectl commands for kubernetes.io
```suggestion
``$ oc describe clusterrole.rbac``
```

File: content/en/docs/reference/labels-annotations-taints/_index.md (hunk 1 above)
Comment (NitishKumar06) [reply content/en/docs/reference/labels-annotations-taints/_index.md:121]: Do you have any idea from where I can learn about this annotation?

Comment (sftim) [reply content/en/docs/reference/labels-annotations-taints/_index.md:121]: - https://github.com/kubernetes/kubernetes has the source code
- you could ask SIG Auth folks
- it's also OK to leave this for someone who is already very familiar with RBAC in Kubernetes

Comment (liggitt) [reply content/en/docs/reference/labels-annotations-taints/_index.md:121]: a couple tweaks needed:

1. this is a label, not an annotation
2. this is added to default Role / ClusterRole / RoleBinding / ClusterRoleBinding objects created by the `kube-apiserver`, not `Cluster` objects

File: content/en/docs/reference/labels-annotations-taints/_index.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (NitishKumar06) [reply content/en/docs/reference/labels-annotations-taints/_index.md:127]: Do you have any suggestions like how should the text go about it? Or any resources that I can refer to put a good text for this annotation? @liggitt

[the remark this answers is not in the document]
Comment (sftim) [reply content/en/docs/reference/labels-annotations-taints/_index.md:127]: As people writing the docs, we can look at the [upstream code](https://github.com/kubernetes/kubernetes), or we can ask in Slack (SIG Auth would be good folks to talk to).

I recommend _against_ mentioning specific contributors with requests like this. The chances are that you're adding work to someone who is more-than-average visible and who faces more-than-average requests for bits of help. A more inclusive option is to see if there's a contributor to that SIG who has the time / capacity to work this out.

Personally, I'd be wary of opening a PR to add documentation for something that I plain didn't know how to learn about. That's also a good rule of thumb: if you're not sure how you'd find out how something works, think twice before starting to document it.

**Comment by Sea-n** [comment]
Hey @hi-rustin,

Thanks for purposing this PR. But since you didn't respond for a while, we have to close this PR now.
/close

Feel free to `/reopen` when you are ready to move forward.
(Note that please don't push anything while PR closed, or you won't be able to reopen it.)

File: content/en/docs/reference/labels-annotations-taints/_index.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (NitishKumar06) [reply content/en/docs/reference/labels-annotations-taints/_index.md:127]: Hmm I see.

**Pull request closed without merging**